### PR TITLE
fix: button to primary

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes

- from nimrods request https://dailydotdev.slack.com/archives/C069XNYMDGV/p1726839965150909

### Preview domain
https://fix-button.preview.app.daily.dev

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Just inform design so we dont have to change it next week and nobody knows why it was different

**Comment by nensidosari** [comment]
Converted to draft until we get approval by desgin

**Review by ilasw — APPROVED** [review]

**Pull request merged**
